deploy.sh

```diff
@@ -50,9 +50,6 @@ function preprocess_and_publish {
     abort "git rev-parse returned an invalid shorthash from branch '${SOURCE_BRANCH}'"
   fi
 
-  # commit to destination branch with shorthash in message
```

Comment:
Since we're no longer modifying any files (specifically, replacing the onboarding script `src`), there's nothing to commit, and this always fails.